### PR TITLE
add coverage info to unit test output

### PR DESCRIPTION
**Pull request by Jaakko-Os**
-two Makefile targets added
   -coverage statistics written to file
   -code coverage results launched to browser

fix: #283

### CODE REVIEW

**Comment by maelk** [comment]
Can you please link the issue ? Do we want the code coverage to be an independent target or should it be included in the existing targets ?

File: Makefile [hunk 1]
```diff
@@ -46,8 +48,18 @@ travis: test-verbose lint
 unit:
 	go test $(GO_TEST_FLAGS) ./cmd/... ./pkg/...
 
+.PHONY: unit-cover
+unit-cover:
+	go test -coverprofile=cover.out $(GO_TEST_FLAGS) ./cmd/... ./pkg/...
+	go tool cover -func=cover.out -o coverage-stats.out
```
Comment (dhellmann) [inline Makefile]: Ideally we would have a target that reports the stats to the console, so we can run it in CI and see the results in the logs. Is there a separate mode for doing that, or do we just need to cat the output file here?

Comment (Jaakko-Os) [reply Makefile]: If there is no output file defined, the stats will be reported directly to he console. I'll do that.

**Comment by dhellmann** [comment]
This looks good, thanks!

**Pull request merged**